dbcon/joblist/tupleunion.cpp

```diff
@@ -124,20 +126,19 @@ namespace
 
   void normalizeIntToStringWithScale(const Row& in, Row* out, uint32_t i) 
   {
-    ostringstream os;
     double d = in.getIntField(i);
     d /= exp10(in.getScale(i));
-    os.precision(15);
-    os << d;
-    utils::NullString ns(os.str());
+    char buf[datatypes::INT128MAXPRECISION + 1];
```

Comment:
Boilerplate that is cloned or almost cloned 4 times. IMHO should be replaced with a routine.

Reply:
Good point, thanks!